### PR TITLE
Update org viewer

### PR DESCRIPTION
**Pull request by bpierre**

### CODE REVIEW

File: packages/organization-viewer-web/package.json [hunk 1]
```diff
@@ -4,14 +4,20 @@
...
-    "build": "npm run clean && npm run compile",
-    "clean": "rm -rf ./dist",
+    "dev": "yarn clean && webpack-dev-server",
+    "build": "yarn clean && webpack",
+    "clean": "rm -rf ./dist && rm -f tsconfig.tsbuildinfo",
     "compile": "tsc -p tsconfig.json"
   },
   "dependencies": {
-    "plumbery-react": "*"
+    "@emotion/core": "^10.0.28",
```
Comment (Evalir) [inline packages/organization-viewer-web/package.json:13]: 👀👀

**Review by Evalir — COMMENTED** [review]
Wild emotion appears!

File: packages/organization-viewer-web/package.json (hunk 1 above)
Comment (bpierre) [reply packages/organization-viewer-web/package.json:13]: haha yes, I wanted to test its JSX pragma for `css`: https://emotion.sh/docs/css-prop#jsx-pragma

Other than that it’s basically styled-components :smile:

Comment (Evalir) [reply packages/organization-viewer-web/package.json:13]: Haha yeah, I've used emotion before, this option looks cool for inline styling as well! 😃

**Review by 0xGabi — APPROVED** [review]

**Pull request merged**